dockerfiles/alpine_3.5_1.x

```diff
@@ -93,6 +93,8 @@ RUN set -x \
     && git -C /usr/src/tarantool checkout "$TARANTOOL_VERSION" \
     && git -C /usr/src/tarantool submodule update --init --recursive \
     && (cd /usr/src/tarantool; \
+       echo "WARNING: Temporary fix for test/unit/cbus_hang test" ; \
```

Comment:
The issue was resolved here - https://github.com/tarantool/tarantool/commit/d7fa6d34ab4e0956fe8a80966ba628e0e3f81067

Is this change still relevant?

Reply:
Unfortunately yes, we tagged some releases before the fix, fortunately this is the patch for the additional test which is not needed for Tarantool in real.